api/Controllers/pageController.js

```diff
@@ -0,0 +1,72 @@
+import { retrievePage, storePage } from "../Utils/fileUtils.js";
+import { HttpStatusCodes } from "../Utils/httpStatusCodes.js";
+
+export const getPage = async (req, res) => {
+    var query = 'SELECT * from space_pages_view WHERE organization_name = $1 AND space_name = $2 AND folder_name = $3 AND page_name = $4';
+    var params = [req.params.orgName, req.params.spaceName, req.params.folderName, req.params.pageName];
```

Comment:
Using object destructuring here would be much cleaner i.e 
{ orgName, spaceName, folderName, pageName } = req.params

https://www.tutorialspoint.com/javascript/javascript_object_destructuring.htm